website/docs/cloud-docs/api-docs/team-tokens.mdx

```diff
@@ -38,7 +38,267 @@ description: >-
 
 # Team tokens API reference
 
-Team API tokens grant access to a team's workspaces. Each team can have an API token that is not associated with a specific user. You can create and delete team tokens and list an organization's team tokens.
+Team API tokens grant access to a team's workspaces. Teams are not limited to a single token, and can have multiple tokens at a time. Team tokens are not are not associated with a specific user.
+
+Teams relying on the [**legacy**](/terraform/cloud-docs/api-docs/team-tokens#legacy-team-tokens-api-reference) team token API (`/teams/authentication-token`), can only create a **single**, valid token at a time. Generating a new legacy token when one already exists for the team revokes the existing legacy token, replacing it with a new team token.
+
+You can create and delete team tokens and list an organization's team tokens.
+
+## Generate a new team token
+
+Generates a new team token.
+
+| Method | Path                                 |
+| :----- | :----------------------------------- |
+| POST   | /teams/:team_id/authentication-tokens |
+
+This endpoint returns the secret text of the new authentication token. You can only access the secret when you create it and cannot recover it later.
+
+### Parameters
+
+- `:team_id` (`string: <required>`) - specifies the team ID for generating the team token
+
+### Request body
+
+This POST endpoint requires a JSON object with the following properties as a request payload.
+
+
+| Key path                      | Type   | Default | Description                                                                                                     |
+| ----------------------------- | ------ | ------- | --------------------------------------------------------------------------------------------------------------- |
+| `data.type`                   | string |         | Must be `"authentication-tokens"`.                                                                               |
+| `data.attributes.description` | string |         | The description of the team token. Each description **must** be unique within the context of the team.          |
+| `data.attributes.expired-at`  | string | `null`  | The UTC date and time that the Team Token will expire, in ISO 8601 format. If omitted or set to `null` the token will never expire. |
+
+### Sample payload
+
+```json
+{
+  "data": {
+    "type": "authentication-tokens",
```

Comment:
Updated this to be the plural `authentication-tokens` per [this](https://hashicorp.slack.com/archives/C08FHA29X4Z/p1745425722676109?thread_ts=1745355247.136919&cid=C08FHA29X4Z) Slack comment!